composer.json

```
@@ -7,7 +7,7 @@
         "drupal/chosen_lib": "2.6.0",
         "drupal/clamav": "1.1.0",
         "drupal/console": "^1.0.2",
-        "drupal/fast_404": "^1.0@alpha",
+        "drupal/fast_404": "^2.0@alpha",
```

```suggestion
        "drupal/fast_404": "^2.0",
```

Let's drop the alpha constraint so we can pickup stable when available?